src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -197,15 +198,27 @@ public function sendPhpResponse($exception)
      * Creates the error Response associated with the given Exception.
      *
      * @param \Exception|FlattenException $exception An \Exception instance
+     * @param ResponseInterface           $response  An optional prototype PSR-7 response
      *
-     * @return Response A Response instance
+     * @return Response|ResponseInterface A response instance
      */
-    public function createResponse($exception)
+    public function createResponse($exception, ResponseInterface $response = null)
```

Comment:
It looks really weird to pass a Response here, especially because it's an immutable one.